src/lib/flows/top-up-flow/enter-amount.svelte

```diff
@@ -100,7 +100,7 @@
 <StepLayout>
   <EmojiAndToken emoji="💰" tokenAddress={tokenInfo.info.address} animateTokenOnMount />
   <StepHeader
-    headline={`Add ${tokenInfo?.info.symbol ?? ''} funds`}
+    headline={`Add ${tokenInfo?.info.symbol}`}
```

Comment:
Is there a reason we had a default `''` value before?

Reply:
Unclear to me, but the changes match what we have in Withdraw modal (no ' ' fallback)